.circleci/docker/ubuntu-cuda/Dockerfile

```diff
@@ -15,6 +15,10 @@ ARG EC2
 ADD ./common/install_base.sh install_base.sh
 RUN bash ./install_base.sh && rm install_base.sh
 
+ARG CLANG_VERSION
+ADD ./common/install_clang.sh install_clang.sh
+RUN bash ./install_clang.sh && rm install_clang.sh
+
```

Comment:
Just curious, how clang installation worked before? And do you mind collocating `Install GCC` and `Instal clang` sections next to each other

Reply:
We didn't have clang installed in CUDA images before as nvcc doesn't play well with clang. 
And in this case we only need it to build with XLA GPU support.